src/lib/messages.ts

```diff
@@ -71,6 +71,7 @@ export const messages = {
         pmdConfigNotFoundGenerator: (file: string) => `PMD custom config file couldn't be located. [${file}]. Check Salesforce Code Analyzer > PMD > Custom Config settings`,
         sfMissing: "To use this extension, first install Salesforce CLI `sf` or `sfdx` commands.",
         sfdxScannerMissing: "To use this extension, first install `@salesforce/sfdx-scanner`.",
+		codeAnalyzerMissing: "**To use this extension, first install `code-analyzer` by running `sf install code-analyzer@latest-beta`.**",
```

Comment:
```suggestion
		codeAnalyzerMissing: "**To use this extension, first install the `code-analyzer` Salesforce CLI plugin by running `sf plugins install code-analyzer` in the VS Code integrated terminal.**",
```

Reply:
Technically, `latest` should be fine.

Reply:
Which is to say, technically it should be fine to not use a tag.